console/src/components/CompactHeader/CompactHeader.less

```diff
@@ -0,0 +1,13 @@
+.compactHeader {
+    // font-family: SourceSansPro;
+    .title {
+        font-size: 20px;
+        letter-spacing: 12.5px;
+        color: #ffffff;  
+        line-height: 45px;
+        padding-left: 20px;
+    }
+    .btn {
+        float: right;
```

Comment:
Using float can work here but I don't think it's really what we mean. I think reserving the space is more appropriate since we don't need subsequent content to flow around it. Flex layout can help address the alignment which will work more easily with RTL if we later decide to swap their sides for those locales. Using `ui/Layout` in the component is likely the easiest way to achieve it.